### ISSUE TITLE
[Documentation] Please mention that cron script requires particular EC2 permissions  

### ISSUE THREAD

**Issue by leonidr** [TheDMSGroup/mautic-eb#3]
What type of report is this:

| Q  | A
| ---| ---
| Feature request? |  

## Description:
aws-eb-cron.sh used for running cron tasks calls AWS CLI : aws ec2 describe-instances.
This command requires non default permission to be assigned to aws-elasticbeanstalk-ec2-role.
Read more here: https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/iam-instanceprofile.html

You can add AmazonEC2ReadOnlyAccess policto to aws-elasticbeanstalk-ec2-role for script to work.

**Comment by heathdutton** [comment]
Strange I didn't encounter this! Would it be better if we just made it so that the script doesn't use `aws ec2 describe-instances`?  There are a couple ways around that.

**Comment by leonidr** [comment]
Started to question myself : "Maybe I customized the policies in some non-default way?"
Read https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/iam-instanceprofile.html again - the policies described there do not have required permissions.

@heathdutton , regarding your question, IMHO this project targets very technologically savvy people who will have no problem configuring required permissions. 
I've found alternative solution here (https://stackoverflow.com/a/32795107) but it looks like it requires similar permissions.
Bottom line, it is your call, because you might spend a work week debugging alternative solution while none of your actual users will appreciate the small difference in configuration.
